lib/cpp/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ if(WITH_BOOSTTHREADS)
 else()
   find_package(Boost 1.53.0 REQUIRED)
 endif()
-include_directories(SYSTEM "${Boost_INCLUDE_DIRS}")
```

Comment:
I don't need to remove SYSTEM to fix the cmake build on Mac OS X. On the other hand, I don't see why SYSTEM is needed :-) In any case, this change can be reverted.